modyn/selector/internal/selector_strategies/downsampling_strategies/rho_loss_downsampling_strategy.py

```diff
@@ -18,7 +18,6 @@
 class RHOLossDownsamplingStrategy(AbstractDownsamplingStrategy):
 
     IL_MODEL_STORAGE_STRATEGY = ModelStorageStrategyConfig(name="PyTorchFullModel")
-    IL_MODEL_DUMMY_SELECTION_STRATEGY = "{}"
```

Comment:
Realized I shouldn't purely use a dummy place holder for the selection strategy on the rho pipeline. Because this value decides if we downsample data when training the IL model. So we should fill it with a selection strategy with no downsampling. (Also, it should be a schema-conforming config, because `selector_manager.py` will instantiate the strategy.)